<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Personal License Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current SE_SERVER: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_reg_value">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_licensename"/>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_licensename">
       <property name="text">
        <string>I have a License Server Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_delete">
       <property name="text">
        <string>Delete SE_Server</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_licensefile">
       <property name="text">
        <string>I have a License File</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_licensefile"/>
       </item>
       <item>
        <widget class="QPushButton" name="button_browse">
         <property name="text">
          <string>Browse..</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Help">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Exit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
